docs/src/howto-guides/020_monitor_tedge_health.md

```diff
@@ -1,29 +1,38 @@
 # How to monitor health of tedge daemons
 
-The health of tedge daemon processes like `tedge-mapper`, `tedge-agent` etc can be monitored via MQTT.
+The health of tedge daemons like `tedge-mapper`, `tedge-agent` etc can be monitored via MQTT.
 These daemons expose MQTT health endpoints which you can query to check if the process is still active or not.
 
-The health endpoints conform to the following topic scheme, listening for health check requests:
+To get the last known health status of a daemon you can subscribe to the following topic
 
-`tedge/health-check/<tedge-daemon-name>`
+```
+tedge/health/<tedge-daemon-name>
+```
+
+To refresh the health status of the daemon, publish an empty message on the topic below.
 
-expecting empty messages, triggering the health check.
+```
+tedge/health-check/<tedge-daemon-name>
+```
 
-The daemon will then respond back on the topic:
+> Note: if the response is not received then mostly likely the daemon is down, or not responding
```

Comment:
I would add that if the process stops or crashes or is killed, then a down message is posted and will be retained till the process is re-launched.

E.g the mapper being killed:

```
tedge mqtt sub 'tedge/health/#'

INFO: Connected
[tedge/health/mosquitto-c8y-bridge] 1
[tedge/health/tedge-mapper-c8y] {"pid":51367,"status":"down"}
[tedge/health/tedge-agent] {"pid":13280,"status":"up","time":1675330667}
```

Reply:
added